app/container/template.hbs

```diff
@@ -6,7 +6,9 @@
   </div>
   <div class="right-buttons">
     {{badge-state model=model}}
-    {{action-menu model=model showPrimary=false classNames="ml-10 inline-block" size="sm"}}
```

Comment:
Why?  There's still actions on removed things, like view in api.  If there's wrong things in the menu then the availableActions is wrong for the resource.